superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -1069,7 +1082,9 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
                   </StyledAlertMargin>
                 )}
                 <DatabaseConnectionForm
+                  isPublic={isPublic}
                   db={db}
+                  setPublicSheets={setPublicSheets}
```

Comment:
Nit, since these two are related let's keep them together:

```suggestion
                  db={db}
                  isPublic={isPublic}
                  setPublicSheets={setPublicSheets}
```